config/default_configs/default_config.yml

```diff
@@ -31,11 +31,11 @@ z_max:
   help: "Model top height. Default: 30km"
   value: 30000.0
 kappa_4_vorticity:
-  help: "Hyperdiffusion parameter for vorticity"
-  value: 2.0e17
+  help: "Hyperdiffusion coefficient for vorticity"
+  value: 1.5
 kappa_4_tracer:
-  help: "Hyperdiffusion parameter for tracer"
-  value: 2.0e17
+  help: "Hyperdiffusion coefficient for tracer"
```

Comment:
It would be better to call this "for scalar" (with `_scalar`) in variable name, since it's also applied to energy, which is not considered a tracer.